### ISSUE TITLE
Decimal histograms

### ISSUE THREAD

**Issue by njss** [elastic/elasticsearch#16474]
I am not finding any good reference about how to set up a range histogram using decimal bins.
I could find a plugin that implements this (until version 1.XX) but I am worried with the sustainability of this fix.
Any new plans?

**Comment by jpountz** [comment]
Duplicate of #4847.

Nothing new unfortunately. Aggregations are currently undergoing a significant refactoring that aims at making them parsed on the coordinating node instead of only on the shards. Once it is finished, I think we should look into this feature again.

**Comment by njss** [comment]
Ok, thank you for the answer.
This is considered a very important feature, right!?
